pkg/subctl/benchmark/throughput.go

```diff
@@ -9,7 +9,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func StartThroughputTests(intraCluster bool) {
+var benchmark_verbose bool
```

Comment:
I think we should only have  global var for this.  I suggest renaming this to `Verbose` and reference it in _pkg/subctl/cmd/benchmark.go_ and remove the `verbose` var there. Then no need to pass it around.